compose/cli-command-compatibility.md

```diff
@@ -0,0 +1,46 @@
+---
+description: Docker Compose command compatibility with docker-compose
+keywords: documentation, docs, docker, compose, containers
+title: Docker Compose command compatibility with docker-compose
+---
+
+In the current state, the compose command in the Docker CLI supports most of the docker-compose commands and flags. It is expected to be a drop in replacement for docker-compose. However, there are a few remaining flags that have not been implemented yet. We are prioritizing the implementation of those based on usage metrics and user feedback.
+
+If you see some compose functionality that is not available in the compose command, please [let us know](https://github.com/docker/compose-cli/issues){:target="_blank" rel="noopener" class="_"} so we can prioritize it.
+
+## Commands or flags not yet implemented
+
+The following sub commands are not implemented nor priroitized today. We may add them to our backlog based on user feedback.
```

Comment:
The following commands have not been implemented yet, and maybe implemented at a later time.  Please let us know if these commands are a higher priority for your usecases